proto/api/v1/workspace_setting_service.proto

```diff
@@ -115,6 +115,8 @@ message WorkspaceMemoRelatedSetting {
   string default_visibility = 9;
   // reactions is the list of reactions.
   repeated string reactions = 10;
+  // disable_markdown_keymap disallow the registration of markdown keymap.
+  bool disable_markdown_keymap = 11;
```

Comment:
How about `disable_keyboard_shortcuts`?

<img width="932" alt="image" src="https://github.com/user-attachments/assets/29c67805-bb90-461d-91f0-88a5fc8ae9fa">

https://code.visualstudio.com/docs/getstarted/keybindings#_keyboard-shortcuts-editor

Reply:
Make sense. How about `disable_markdown_keyboard_shortcuts` as it only suppress the markdown related shortcuts(Yeah it does not affect the command+enter to submit)?

Reply:
Then it's recommand to use `disable_markdown_shortcuts` which is simpler and easier to understand.